conda/activate.py

```diff
@@ -226,7 +226,12 @@ def hook(self, auto_activate_base: bool | None = None) -> str:
     def execute(self):
         # return value meant to be written to stdout
         self._parse_and_set_args(self._raw_arguments)
-        return getattr(self, self.command)()
+
+        # invoke pre/post commands, see conda.cli.conda_argparse.do_call
+        context.plugin_manager.invoke_pre_commands(self.command)
+        response = getattr(self, self.command)()
+        context.plugin_manager.invoke_post_commands(self.command)
```

Comment:
Since errors during activation might create a catch 22, e.g. not being able to activate the conda base env with a broken plugin installed, I think we should catch exceptions here, print them and then move on

Reply:
activating `base` is not necessary to call `install` or `remove`

Reply:
and the user can still use the `CONDA_NO_PLUGINS=True` fallback

Reply:
> and the user can still use the `CONDA_NO_PLUGINS=True` fallback

Ooooh, gooood point!